app/javascript/stylesheets/home-page/_backgrounds/home-page_backgrounds.scss

```diff
@@ -0,0 +1,15 @@
+.home-page_background_action-movies {
+  background: no-repeat center/100% url("../../../images/bg_action_movies001.png");
```

Comment:
1) Why complicate the names of folders and classes so much? БЭМ is not about that. And in a small project it will only complicate
2) app/assets/stylesheets/home/index (show.....)
3) 
.home_page_wrap {
  .western_bg {
  }
}
2)asset-url('bg_action_movies001.png')